src/docs/config.mdx

```diff
@@ -113,9 +110,9 @@ LOGGING['default_level'] = 'WARNING'
 
 If logging in a particular module is not showing up when you expect it to, you should check the log level for that module in `src/sentry/conf/server.py` in the `LOGGING` variable.
 
-<Note>
-If you are a Sentry employee who is looking for logs in Kibana and they aren't showing up when you expect them to, one potential reason is that a field in the log message does not match Elasticsearch's schema. See the index patterns <a href="https://kibana.getsentry.net/app/kibana#/management/kibana/indices/0ba261d0-97ca-11e9-90db-dfacda4da93b?_g=()&_a=(tab:indexedFields)">here</a>.
-</Note>
+<Note><markdown>
```

Comment:
I haven't found it necessary to use the <markdown></markdown> elements in user docs, FYI, but I don't think they'll cause issues.

Reply:
Eh, will leave them for now. May do another sweep to fix them.